marketplace-ui/src/app/modules/product/product-detail/product-detail-feedback/product-star-rating-panel/add-feedback-dialog/add-feedback-dialog.component.html

```diff
@@ -27,7 +27,7 @@ <h5 class="text-secondary">
       <p class="text-secondary mb-2">
         {{ 'common.feedback.starRatingTitle' | translate }}
       </p>
-      <app-star-rating [rate]="feedback.rating" (rateChange)="onRateChange($event)" [starClass]="'adding-feedback-star'" />
+      <app-star-rating [rate]="feedback.rating" (rateChange)="onRateChange($event)" [starClass]="'adding-feedback-star'" [ratingStarsClass]="'rating-stars'" />
```

Comment:
why need this input

Reply:
This input need for the component app-star-rating if it display in add-feedback-dialog. The width 40% will apply for dialog in case view port is larger than 768px and 100% for the opposite